enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -57,30 +57,37 @@
     public KernelTransaction tx;
 
     @Procedure( name = "dbms.security.createUser", mode = DBMS )
-    public void createUser( @Name( "username" ) String username, @Name( "password" ) String password,
-            @Name( "requirePasswordChange" ) boolean requirePasswordChange )
+    public void createUser(
+            @Name( "username" ) String username,
+            @Name( "password" ) String password,
+            @Name( value = "requirePasswordChange", defaultValue = "true"  ) boolean requirePasswordChange
+    )
             throws InvalidArgumentsException, IOException
     {
         EnterpriseAuthSubject adminSubject = ensureAdminAuthSubject();
         adminSubject.getUserManager().newUser( username, password, requirePasswordChange );
     }
 
     @Procedure( name = "dbms.security.changeUserPassword", mode = DBMS )
-    public void changeUserPassword( @Name( "username" ) String username, @Name( "newPassword" ) String newPassword )
+    public void changeUserPassword(
+            @Name( "username" ) String username,
+            @Name( "newPassword" ) String newPassword,
+            @Name( value = "requirePasswordChange", defaultValue = "true" ) boolean requirePasswordChange
+    )
             throws InvalidArgumentsException, IOException
     {
         EnterpriseAuthSubject enterpriseSubject = EnterpriseAuthSubject.castOrFail( authSubject );
         if ( enterpriseSubject.doesUsernameMatch( username ) )
         {
-            enterpriseSubject.setPassword( newPassword );
+            enterpriseSubject.setPassword( newPassword, false );
```

Comment:
If the user passes a true here, should we not use that, or alternatively throw an exception if we think it too irrational to set your own password with a require password change? I think we should just do what the user says. I can imagine a user wanting to do this.